src/screens/dashboard/dashboard.tsx

```diff
@@ -56,7 +56,7 @@ const mapStateToProps = (state: IReduxState) => ({
     blockchains: getBlockchains(state),
     selectedBlockchain: getSelectedBlockchain(state),
     selectedAccount: getSelectedAccount(state),
-    exchangeRates: (state as any).market.exchangeRates,
+    exchangeRates: state.market.exchangeRates,
```

Comment:
are exchange rates needed on dashboard ?

Reply:
![image](https://user-images.githubusercontent.com/32679726/75314417-5d193e80-5868-11ea-88e1-3fd2567b1df1.png)

`exchangeRates` are being used on Dashboard in order to `calculateBalance` for `CoinBalanceCard`

Reply:
good, that's fine :)
but on the screenshot i see a problem : `toCurrency="USD"`, please change it, use the value from redux

Reply:
ok sure, fixing this